src/libs/scene_utils/README.md

```diff
@@ -64,7 +64,99 @@ You will need to finish the Getting Started sample [setup](https://github.com/aw
 
 ---
 
-## 2. DELETE `src/delete_sample/`
+## 2. CESIUM `samples/cesium_sample/sample.ts`
+
+Automates the Cesium Ion pipeline to convert 3D assets to 3D Tiles and upload directly to IoT TwinMaker.
+
+1. Upload your 3D asset to Cesium and wait for it to perform the conversion to a 3D Tiles tileset.
+2. Create an archive for the tileset to prepare for download.
+3. Download archive for tileset.
+4. Upload tileset to your TwinMaker workspace S3 bucket.
+5. Create/edit a TwinMaker scene with the 3D tileset.
+
+### Prerequisites
+
+1. Register Cesium and choose the appropriate [subscription plan](https://cesium.com/platform/cesium-ion/pricing/) based on your business requirement
+2. Create a Cesium asset token
+
+   a. Go to the [tokens page](https://cesium.com/ion/tokens)
+
+   b. Click “Create token”
+
+   c. Name it “TwinMaker token”
+
+   d. Toggle on the following permission scopes: 1. assets:list 2. assets:read 3. assets:write 4. archives:read 5. archives:write
+
+   e. Click “Create”
+
+   f. Set the environment variable CESIUM_ACCESS_TOKEN with this token
+
+### Example
+
+This script runs up to 5 steps to convert your 3D asset into 3D Tiles for use in an IoT TwinMaker scene. When you start on any example below, the script will continue with all 5 steps. For example, if you have an asset already uploaded to Cesium Ion, start on step 2 to archive, download, and add the tileset to your scene.
+
+1. Upload a 3D asset to Cesium
+
+```bash
+npx ts-node samples/cesium_sample/sample.ts --workspaceId [WORKSPACE_ID] --sceneId [SCENE_ID] --assetFilePath [3D_ASSET_PATH] --dracoCompression
```

Comment:
fyi that I'm getting this error when I try to run it, I haven't done anything in Cesium besides create the token and export it

```
% npx ts-node samples/cesium_sample/sample.ts --workspaceId [redacted] --sceneId [non-existent] --assetFilePath ~/Downloads/[300MB GLB file] --dracoCompression
(node:38119) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.

Please migrate your code to use AWS SDK for JavaScript (v3).
For more information, check the migration guide at https://a.co/7PzMCcy
(Use `node --trace-warnings ...` to show where the warning was created)
Uploading created scene file to [workspace-bucket redacted]
Uploading asset to Cesium Ion...
Error: [Error 404] Failed to process asset upload request with error: Not Found
    at CesiumClient.checkStatus (/private/tmp/aws-iot-twinmaker-samples/src/libs/scene_utils/client/cesium.ts:16:17)
    at CesiumClient.handleRequest (/private/tmp/aws-iot-twinmaker-samples/src/libs/scene_utils/client/cesium.ts:22:17)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async CesiumClient.handleCesiumAPI (/private/tmp/aws-iot-twinmaker-samples/src/libs/scene_utils/client/cesium.ts:41:22)
    at async CesiumClient.upload (/private/tmp/aws-iot-twinmaker-samples/src/libs/scene_utils/client/cesium.ts:188:30)
    at async Object.exports.uploadAssetForTiling (/private/tmp/aws-iot-twinmaker-samples/src/libs/scene_utils/samples/cesium_sample/sample_utils.ts:167:29)
    at async /private/tmp/aws-iot-twinmaker-samples/src/libs/scene_utils/samples/cesium_sample/sample.ts:38:31
```